testing/test_terminal.py

```diff
@@ -823,6 +823,27 @@ def pytest_report_header(config, startdir):
             str(testdir.tmpdir),
         ])
 
+    def test_show_capture(self, testdir):
```

Comment:
We also need to test `both` (explicitly) and `no`.

Reply:
fixed